app/models/gbif_genbank_linker.rb

```diff
@@ -180,30 +194,32 @@ def each
     return to_enum(:each) unless block_given?
 
     File.open(@gbif) do |gbifio|
-      File.open(@genbank) do |genbankio|
+      file_clazz = genbank_gz? ? Zlib::GzipReader : File
+      file_clazz.open(@genbank) do |genbankio|
```

Comment:
This adds `gz` support for genbank files. From now on, we'll keep them as gz files on the disk and de-compress them in memory.